frontend/helpers/dashboard.ts

```diff
@@ -0,0 +1,32 @@
+import { useRouter } from 'next/router';
```

Comment:
I've added the `useSortChange` helper from #264 which can be seen in use [here](https://github.com/Vizzuality/heco-invest/pull/264/files#diff-247b86e99b57820e5f5c42c14de2ba95e310bd0c726a00c12ae6dd8ff6045cd3), to help unblock other dashboard tasks.